packages/tooling/cfp-template/template/src/components/welcome/welcome.pw.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { expect, test } from '@playwright/test';
```

Comment:
When I was playing around with things a little while back I was able to use Playwright with Chai. What about using that configuration so it matches the rest of our tests? The setup I used was from `modern-web.dev` https://modern-web.dev/guides/test-runner/getting-started/

Reply:
Is that our position for the future when we switch off Karma?

Reply:
I'm fine with doing mocha chai as well since presumably we can run unit tests with just that and also use it for assertion in playwright for homogeny like you mentioned, but I also don't want to unilaterally decide. Let's get opinions from @nicholasrice and @chrisdholt at minimum. Responding here might be best as it's in context.

Reply:
I took the path of least resistance for the SSR package, which is using `expect` from Playwright. It has been suitable so far, though I haven't flexed it in that package. I believe under the hood, Playright exposes Jest's `expect` (https://playwright.dev/docs/api/class-test) though, so I expect it is robust. I personally like Jest's `expect` and find it intuitive, but I can appreciate if it's not everyone's preference. As long as IntiliSense works, I'm comfortable working with any assertion lib.